src/locales/en-US/commands.json

```diff
@@ -362,6 +362,44 @@
     "invalidMathExpression": "Invalid math expression.",
     "result": "Result: `{{result}}`"
   },
+  "osu": {
+    "commandDescription": "Searches for a profile or a beatmap on osu!",
+    "commandUsage": "<type> <query>",
+    "subcommands": {
+      "player": {
+        "commandDescription": "Searches for a profile on osu!",
+        "commandUsage": "[profile name or id]",
+        "noPlayer": "You have to give me a valid osu! profile name or ID.",
+        "level": "Level **{{number}}**",
+        "joinedAt": "Joined on **{{date}}** ({{timeAgo}})",
+        "globalRanking": "Global Ranking: **#{{rank}}**",
+        "countryRanking": "Country Ranking: **#{{rank}}**",
+        "rankedScore": "Ranked Score: **{{rankedScore}}**",
+        "playTime": "Played for **{{time}}** - **{{pp}}** pp",
+        "totalScore": "Total Score: **{{totalScore}}**",
+        "hitAccuracy": "Hit Accuracy: **{{accuracy}}%**",
+        "playCount": "Play Count: **{{playCount}}**",
+        "totalHits": "Total Hits: **{{totalHits}}** (**{{300}}** {{emoji300}}, **{{100}}** {{emoji100}}, **{{50}}** {{emoji50}})",
+        "topScores": "**{{user}}**'s top scores:",
+        "recentPlays": "**{{user}}**'s most recent plays in the last 24 hours:",
+        "playerNotFound": "Profile not found."
+      },
+      "beatmap": {
+        "commandDescription": "Searches for a beatmap or beatmap set on osu!",
+        "commandUsage": "<beatmap or beatmap set id>",
+        "noBeatmap": "You have to give me a valid beatmap or beatmap set ID.",
+        "starDifficulty": "Star Difficulty: **{{difficulty}}**",
+        "successRate": "Success Rate: **{{rate}}%** ({{successfulPlays}} / {{totalPlays}})",
+        "mappedBy": "Mapped by {{mapper}}",
+        "lastUpdate": "Last update on **{{date}}** ({{timeAgo}})",
+        "approved": "Approved on **{{date}}** ({{timeAgo}})",
+        "tags": "Tags: **{{tags}}**",
+        "download": "[Download]({{link}}) - [no video]({{linkNoVideo}})",
```

Comment:
```suggestion
        "download": "[Download]({{link}}) - [Download without video]({{linkNoVideo}})",
```

Reply:
What happens if a beatmap already has no video?

Reply:
It'll download just fine, but I can make that check if it has or hasn't got a video

Reply:
Please do :)